Traceback (most recent call last):
  File "/opt/hostedtoolcache/Python/3.8.12/x64/lib/python3.8/site-packages/jupyter_cache/executors/utils.py", line 51, in single_nb_execution
    executenb(
  File "/opt/hostedtoolcache/Python/3.8.12/x64/lib/python3.8/site-packages/nbclient/client.py", line 1085, in execute
    return NotebookClient(nb=nb, resources=resources, km=km, **kwargs).execute()
  File "/opt/hostedtoolcache/Python/3.8.12/x64/lib/python3.8/site-packages/nbclient/util.py", line 84, in wrapped
    return just_run(coro(*args, **kwargs))
  File "/opt/hostedtoolcache/Python/3.8.12/x64/lib/python3.8/site-packages/nbclient/util.py", line 62, in just_run
    return loop.run_until_complete(coro)
  File "/opt/hostedtoolcache/Python/3.8.12/x64/lib/python3.8/asyncio/base_events.py", line 616, in run_until_complete
    return future.result()
  File "/opt/hostedtoolcache/Python/3.8.12/x64/lib/python3.8/site-packages/nbclient/client.py", line 551, in async_execute
    await self.async_execute_cell(
  File "/opt/hostedtoolcache/Python/3.8.12/x64/lib/python3.8/site-packages/nbclient/client.py", line 846, in async_execute_cell
    self._check_raise_for_error(cell, exec_reply)
  File "/opt/hostedtoolcache/Python/3.8.12/x64/lib/python3.8/site-packages/nbclient/client.py", line 748, in _check_raise_for_error
    raise CellExecutionError.from_cell_and_msg(cell, exec_reply_content)
nbclient.exceptions.CellExecutionError: An error occurred while executing the following cell:
------------------
from nltk.tokenize import sent_tokenize, word_tokenize
from nltk import pos_tag
from nltk import ne_chunk

sentences = sent_tokenize(phrase1)

for sentence in sentences:
    print("Sentence:", sentence)
    words = word_tokenize(sentence)
    pos_tags_sentence = pos_tag(words)
    ne_chunks_sentence = ne_chunk(pos_tags_sentence)
    print(ne_chunks_sentence)
------------------

[0;31m---------------------------------------------------------------------------[0m
[0;31mLookupError[0m                               Traceback (most recent call last)
Input [0;32mIn [6][0m, in [0;36m<module>[0;34m[0m
[1;32m      9[0m words [38;5;241m=[39m word_tokenize(sentence)
[1;32m     10[0m pos_tags_sentence [38;5;241m=[39m pos_tag(words)
[0;32m---> 11[0m ne_chunks_sentence [38;5;241m=[39m [43mne_chunk[49m[43m([49m[43mpos_tags_sentence[49m[43m)[49m
[1;32m     12[0m [38;5;28mprint[39m(ne_chunks_sentence)

File [0;32m/opt/hostedtoolcache/Python/3.8.12/x64/lib/python3.8/site-packages/nltk/chunk/__init__.py:185[0m, in [0;36mne_chunk[0;34m(tagged_tokens, binary)[0m
[1;32m    183[0m [38;5;28;01melse[39;00m:
[1;32m    184[0m     chunker_pickle [38;5;241m=[39m _MULTICLASS_NE_CHUNKER
[0;32m--> 185[0m chunker [38;5;241m=[39m [43mload[49m[43m([49m[43mchunker_pickle[49m[43m)[49m
[1;32m    186[0m [38;5;28;01mreturn[39;00m chunker[38;5;241m.[39mparse(tagged_tokens)

File [0;32m/opt/hostedtoolcache/Python/3.8.12/x64/lib/python3.8/site-packages/nltk/data.py:868[0m, in [0;36mload[0;34m(resource_url, format, cache, verbose, logic_parser, fstruct_reader, encoding)[0m
[1;32m    865[0m     [38;5;28mprint[39m([38;5;124m'[39m[38;5;124m<<Loading [39m[38;5;132;01m%s[39;00m[38;5;124m>>[39m[38;5;124m'[39m [38;5;241m%[39m (resource_url,))
[1;32m    867[0m [38;5;66;03m# Load the resource.[39;00m
[0;32m--> 868[0m opened_resource [38;5;241m=[39m [43m_open[49m[43m([49m[43mresource_url[49m[43m)[49m
[1;32m    870[0m [38;5;28;01mif[39;00m [38;5;28mformat[39m [38;5;241m==[39m [38;5;124m'[39m[38;5;124mraw[39m[38;5;124m'[39m:
[1;32m    871[0m     resource_val [38;5;241m=[39m opened_resource[38;5;241m.[39mread()

File [0;32m/opt/hostedtoolcache/Python/3.8.12/x64/lib/python3.8/site-packages/nltk/data.py:993[0m, in [0;36m_open[0;34m(resource_url)[0m
[1;32m    990[0m protocol, path_ [38;5;241m=[39m split_resource_url(resource_url)
[1;32m    992[0m [38;5;28;01mif[39;00m protocol [38;5;129;01mis[39;00m [38;5;28;01mNone[39;00m [38;5;129;01mor[39;00m protocol[38;5;241m.[39mlower() [38;5;241m==[39m [38;5;124m'[39m[38;5;124mnltk[39m[38;5;124m'[39m:
[0;32m--> 993[0m     [38;5;28;01mreturn[39;00m [43mfind[49m[43m([49m[43mpath_[49m[43m,[49m[43m [49m[43mpath[49m[43m [49m[38;5;241;43m+[39;49m[43m [49m[43m[[49m[38;5;124;43m'[39;49m[38;5;124;43m'[39;49m[43m][49m[43m)[49m[38;5;241m.[39mopen()
[1;32m    994[0m [38;5;28;01melif[39;00m protocol[38;5;241m.[39mlower() [38;5;241m==[39m [38;5;124m'[39m[38;5;124mfile[39m[38;5;124m'[39m:
[1;32m    995[0m     [38;5;66;03m# urllib might not use mode='rb', so handle this one ourselves:[39;00m
[1;32m    996[0m     [38;5;28;01mreturn[39;00m find(path_, [[38;5;124m'[39m[38;5;124m'[39m])[38;5;241m.[39mopen()

File [0;32m/opt/hostedtoolcache/Python/3.8.12/x64/lib/python3.8/site-packages/nltk/data.py:701[0m, in [0;36mfind[0;34m(resource_name, paths)[0m
[1;32m    699[0m sep [38;5;241m=[39m [38;5;124m'[39m[38;5;124m*[39m[38;5;124m'[39m [38;5;241m*[39m [38;5;241m70[39m
[1;32m    700[0m resource_not_found [38;5;241m=[39m [38;5;124m'[39m[38;5;130;01m\n[39;00m[38;5;132;01m%s[39;00m[38;5;130;01m\n[39;00m[38;5;132;01m%s[39;00m[38;5;130;01m\n[39;00m[38;5;132;01m%s[39;00m[38;5;130;01m\n[39;00m[38;5;124m'[39m [38;5;241m%[39m (sep, msg, sep)
[0;32m--> 701[0m [38;5;28;01mraise[39;00m [38;5;167;01mLookupError[39;00m(resource_not_found)

[0;31mLookupError[0m: 
**********************************************************************
  Resource [93mmaxent_ne_chunker[0m not found.
  Please use the NLTK Downloader to obtain the resource:

  [31m>>> import nltk
  >>> nltk.download('maxent_ne_chunker')
  [0m
  For more information see: https://www.nltk.org/data.html

  Attempted to load [93mchunkers/maxent_ne_chunker/PY3/english_ace_multiclass.pickle[0m

  Searched in:
    - '/home/runner/nltk_data'
    - '/opt/hostedtoolcache/Python/3.8.12/x64/nltk_data'
    - '/opt/hostedtoolcache/Python/3.8.12/x64/share/nltk_data'
    - '/opt/hostedtoolcache/Python/3.8.12/x64/lib/nltk_data'
    - '/usr/share/nltk_data'
    - '/usr/local/share/nltk_data'
    - '/usr/lib/nltk_data'
    - '/usr/local/lib/nltk_data'
    - ''
**********************************************************************

LookupError: 
**********************************************************************
  Resource [93mmaxent_ne_chunker[0m not found.
  Please use the NLTK Downloader to obtain the resource:

  [31m>>> import nltk
  >>> nltk.download('maxent_ne_chunker')
  [0m
  For more information see: https://www.nltk.org/data.html

  Attempted to load [93mchunkers/maxent_ne_chunker/PY3/english_ace_multiclass.pickle[0m

  Searched in:
    - '/home/runner/nltk_data'
    - '/opt/hostedtoolcache/Python/3.8.12/x64/nltk_data'
    - '/opt/hostedtoolcache/Python/3.8.12/x64/share/nltk_data'
    - '/opt/hostedtoolcache/Python/3.8.12/x64/lib/nltk_data'
    - '/usr/share/nltk_data'
    - '/usr/local/share/nltk_data'
    - '/usr/lib/nltk_data'
    - '/usr/local/lib/nltk_data'
    - ''
**********************************************************************


